exercises/introduction-part-2.md

```diff
@@ -0,0 +1,27 @@
+# Introduction - part 2
+
+In this course, you will use _block-based programming_, like on code.org, to create JavaScript that changes the html on a web page.
+
+## Prerequisites
+
+These exercises assume you have tried block based programming already. Do the [code.org course](https://studio.code.org/s/course3) before you start. 
+You should at least have done the lessons up to Conditionals and While Loops.
+
+## Understanding the exercises - rendering HTML
+
+- You can write html in the text area above.
+- Try writing a `p` element in the text area above that contains the text `hello world!`
+- Try clicking the "run" button to **render** the html element in the area below.
+
+> ⚙️ **render** means take the html code and interpret in order to produce a web page. All modern browsers work by interpreting a html document and converting into a web page
```

Comment:
@gregdyke @SallyMcGrath I've split the `introduction.md` text into three sections based on how I imagined it could be broken down. Don't know if someone has already done this but it didn't take long for me to do this so feel free to disregard it if you want.

Reply:
I like it. But until I have a better idea of how we put the .md files back together, I'm not quite sure what that looks like? I'm not sure we want 3 intro exercises before finally moving to the first exercise? But maybe we do :)